homebrew_challenge/src/components/Facts/Facts.css

```diff
@@ -0,0 +1,71 @@
+.icons-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.facts-container {
+  margin: 0;
+  padding: 20px;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  font-size: 14px;
+  background-color: #f2f2f2;
+}
+
+.facts-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.facts-component {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  text-align: left;
+  width: 19%;
+  min-height: 110px;
```

Comment:
This looks a bit weird on my browser - too much whitespace at the bottom on a wide browser. Maybe remove this and just add an extra 5px to the padding-bottom if you need it instead?

Reply:
refactored as suggested